README.md

```diff
@@ -1,213 +1,23 @@
-[![Code Climate](https://codeclimate.com/github/ucsdlib/damspas/badges/gpa.svg)](https://codeclimate.com/github/ucsdlib/damspas)
-
-[![Test Coverage](https://codeclimate.com/github/ucsdlib/damspas/badges/coverage.svg)](https://codeclimate.com/github/ucsdlib/damspas/coverage)
-
-[![Dependency Status](https://gemnasium.com/ucsdlib/damspas.svg)](https://gemnasium.com/ucsdlib/damspas)
+[![CircleCI](https://circleci.com/gh/ucsdlib/damspas.svg?style=svg)](https://circleci.com/gh/ucsdlib/damspas)
+[![Coverage Status](https://coveralls.io/repos/github/ucsdlib/damspas/badge.svg?branch=master)](https://coveralls.io/github/ucsdlib/damspas?branch=master)
 
 [UC San Diego Library](https://library.ucsd.edu/ "UC San Diego Library") Digital Collections Public Access System.
 
 A Hydra repository backed by [DAMS Repository](http://github.com/ucsdlib/damsrepo).
 
 # Docker
-  You can below in Setup Instructions, or you can just do this:
-
-  ``` sh
-    $ ./bin/dc up
-    $ ./bin/dc exec web rake db:setup
-  ```
-
-  Then navigate to localhost:3000
-
-  
-# Setup Instructions 
-
-## Windows
-_Let's not go there._
-
-## Apple/OSX
```

Comment:
Do we want to keep this as reference for setting up local development environment? But it's out dated and we may still need to clean it up bit if we want to keep it though. Thanks.

Reply:
@lsitu - it's a very good question, I'd be interested to hear what others think as well.

In my opinion, it's both outdated and no longer relevant. I think the `docker-compose` setup we have is far preferable than setting up each of the pieces separately, and much of the instructions at the bottom were more deployment/production related, which are documented better elsewhere by/for OPS. But that is just my opinion, if others on the team found value in those docs, perhaps we can work on updating them instead.

Reply:
@mcritchlow It sounds good if we have this kind of deployment/production related setup documented somewhere and link to it from README.